workflows/basemaps/cron-vector-etl.yaml

```
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/argoproj/argo-workflows/v3.4.13/api/jsonschema/schema.json
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: cron-vector-etl
+  labels:
+    linz.govt.nz/category: basemaps
+    linz.govt.nz/data-type: vector
+spec:
+  schedule: '0 06 * * 3'
+  timezone: 'NZ'
```

"NZ" is not a valid time zone name. Based on the schema it looks like it should be this:

```suggestion
  timezone: 'Pacific/Auckland'
```

---

Both NZ and Pacific/Auckland are IANA timezone standard, actually the NZ is links to Pacific/Auckland.

---

TIL, thanks!